### ISSUE TITLE
how can I find training, test and validation data?

### ISSUE THREAD

**Issue by gireeshkbogu** [neheller/kits19#40]
Somehow I downloaded all 300 files in one folder. How can I separate the train set from the test set? Also, any light on how the top teams generated the validation set (was it random split, 70:30 split?) would be greatly helpful? thanks

**Comment by neheller** [comment]
Hi,

You'll find that cases 210 - 299 don't have associated segmentation.nii.gz files. Those belong in the test set. As far as splitting into training and validation, the winning team used five-fold cross validation. You can see what they and others did in more detail at [results.kits-challenge.org/miccai2019](results.kits-challenge.org/miccai2019
)

Cheers,
Nick

**Comment by gireeshkbogu** [comment]
Thank you for the quick response, Nick. This link is not working byw.

**Comment by neheller** [comment]
My apologies, it should work now.
